internal/services/dispatcher/dispatcher.go

```diff
@@ -370,6 +427,29 @@ func (d *DispatcherImpl) handleTask(ctx context.Context, task *msgqueue.Message)
 	return err
 }
 
+func (d *DispatcherImpl) handleV1Task(ctx context.Context, task *msgqueuev1.Message) (err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			recoverErr := recoveryutils.RecoverWithAlert(d.l, d.a, r)
+
+			if recoverErr != nil {
+				err = recoverErr
+			}
+		}
+	}()
+
+	switch task.ID {
+	case "task-assigned-bulk":
```

Comment:
qq: are these values type checked?

Reply:
no, this is a bit of a relic from our old engine -- at some point I'd like to refactor how we're doing message payloads, but not yet.